optuna_dashboard/ts/components/TrialList.tsx

```diff
@@ -226,10 +226,13 @@ const TrialListDetail: FC<{
               ? "rgba(255, 255, 255, 0.05)"
               : "rgba(0, 0, 0, 0.05)",
           width: "100%",
+          maxWidth: "1000px",
```

Comment:
How about removing this for wide display users?

```suggestion
```

Reply:
@c-bata Thanks comment!

Without a maximum width setting, the window will be as wide as the length of the string.
I'm trying to fix a problem with a Human-in-the-loop Slider widget where the width of the slider is lengthened to match the length of the string.(see Horizontal scrolling part of the issue)
So I would like to remove the maximum width in the section you commented on,  and add a maximum value to the slider width.
what do you think?

(Maybe below line?)
https://github.com/optuna/optuna-dashboard/blob/main/optuna_dashboard/ts/components/TrialFormWidgets.tsx#L284

Reply:
The following modifications are intended
![Screenshot 2023-05-16 at 20 13 23](https://github.com/optuna/optuna-dashboard/assets/23289252/98d08d64-bfd5-4ff3-954c-ac869b12a8c6)

Reply:
@hrntsm Thank you for your explanation. Sounds reasonable!